src/logging/LogItem.ts

```diff
@@ -91,12 +154,12 @@ export class LogItem {
      * 
      * Hence, the child item is not returned.
      */
-    log(labelOrValues, logLevel = null) {
-        const item = this.child(labelOrValues, logLevel, null);
-        item._end = item._start;
+    log(labelOrValues: LabelOrValues, logLevel?: LogLevel): void {
+        const item = this.child(labelOrValues, logLevel);
+        item.end = item.start;
     }
 
-    set(key, value) {
+    set(key: string | object, value: unknown): void {
```

Comment:
I think value is optional?
```suggestion
    set(key: string | object, value?: unknown): void {
```